frontend/packages/core/src/NPS/feedback.tsx

```diff
@@ -43,18 +47,26 @@ export const defaults: IClutch.feedback.v1.ISurvey = {
   ],
 };
 
-const StyledButton = styled(Button)({
-  fontSize: "14px",
-  padding: "0 8px",
-  height: "18px",
-});
+const StyledButton = styled(Button)<{ $origin: Origins }>({}, props =>
+  props.$origin === "WIZARD"
```

Comment:
nit: maybe extract this into constant

Reply:
Good idea, I'll pull it out so I can remove the props